### PR TITLE
Add option to disable links

### PR DESCRIPTION
**Pull request by infraredgirl**

### CODE REVIEW

File: less/links.less [hunk 1]
```diff
@@ -0,0 +1,5 @@
+a.disabled {
```
Comment (jtomasek) [inline less/links.less:1]: using
`a.disabled, a[disabled] {`
gives a nice alternative of using an disabled attribute to set the anchor disabled. Although doing so is not html valid, but anyway...:)

File: dist/css/patternfly-additions.css [hunk 2]
```diff
@@ -3325,6 +3325,11 @@ table.datatable th:active {
 .layout-pf-alt.layout-pf-alt-fixed-with-footer body {
   padding-bottom: 37px;
 }
+a.disabled {
+  color: #8b8d8f;
+  cursor: not-allowed;
+  text-decoration: none;
+}
```
Comment (jeff-phillips-18) [inline dist/css/patternfly-additions.css]: I believe we should also turn off the text decorations on hover so the underline is not shown.

**Comment by jeff-phillips-18** [comment]
LGTM

File: less/links.less (hunk 1 above)
Comment (andresgalante) [inline less/links.less:1]: @jtomasek good point but there is no disabled attribute on anchors on html, I'll merge it and we can iterate on it later.

@akurdyukov thanks a lot!

**Pull request merged**